code/modules/research/nodes/illegal.dm

```diff
@@ -45,20 +45,22 @@
 
 	unlocks_designs = list(/datum/design/research/item/chameleon_kit)
 
-/datum/technology/freedom_implant
```

Comment:
just leave those implants for greater good and push changes only for shades pls

Reply:
Validhunting? No, i just add variability to implants and to illegal technologies (because they are in lack of interesting items to use).